examples/platform/asset_groups.py

```diff
@@ -0,0 +1,191 @@
+    """
+    # CBCloudAPI is the connection to the cloud.  It holds the credentials for connectivity.
+    # To execute this script, the profile must have an API key with the following permissions.
+    # If you are restricted in the actions you're allowed to perform, expect a 403 response for missing permissions.
+    # Permissions are set on Settings -> API Access -> Access Level and then assigned to an API Key
+    # GM - Group Management - group-management - CREATE, READ, UPDATE, DELETE: For managing asset groups
+    # Device - General Information - device - READ: For getting device information
+    # Policies - Policies - org.policies - READ: For viewing policy information and pre-viewing the impact of changes
+    #    to policy ranking and asset groups.
+
+    # api = CBCloudAPI(profile="YOUR_PROFILE_HERE")
+    api = CBCloudAPI(profile="ASSET_GROUP_DEV_01")
+
+    # to get all asset groups, a static method is available on the AssetGroup class.
+    # This is useful for listing the groups configured in your org
+    all_asset_groups = AssetGroup.get_all_groups(api)
+    for ag in all_asset_groups:
+        print("asset group name: {}, member count: {}".format(ag.name, ag.member_count))
+
+    # Create an asset group.  The only mandatory field is the Asset Group Name.
+    # It can be created without a policy, which enables the use of group for visibility of specific assets
+    # It can be created without a query, which enables manual assignment of assets to the group later
+    new_asset_group_name = "My Example Asset Group"
+    new_asset_group = AssetGroup.create_group(api, new_asset_group_name, "Demonstrating the SDK")
+    print(new_asset_group)
+
+    # Add a query.  All assets that match this criteria will be dynamically added to the group
+    new_asset_group.query = "os.equals:WINDOWS"
+    # Assign a policy.  All assets in the group may have this policy applied.  If an asset is in more than one group,
+    # policy ranking determines which is the effective policy.
+    bottom_rank_policy = None
+    for p in api.select(Policy).all():
+        if bottom_rank_policy is None or p.position > bottom_rank_policy.position:
+            bottom_rank_policy = p
+    new_asset_group.policy = bottom_rank_policy.id
+    new_asset_group.save()
+    print("\n\n new_asset_group {}".format(new_asset_group))
+    # Clean up after ourselves and delete the asset group
+    new_asset_group.delete()
+
+    # An asset group can also be created with a query and / or a policy included
+    print("\n\n Second asset group with policy and query")
+    second_name = "Second demo group"
+    second_asset_group = AssetGroup.create_group(api, second_name, "Second group description",
+                                                 query="os.equals:MAC", policy_id=bottom_rank_policy.id)
+    second_asset_group.refresh()
+    while second_asset_group.status != "OK":
+        print("waiting")
+        time.sleep(10)
+        second_asset_group.refresh()
```

Comment:
What is this for? I'm not familiar with what the status is indicating

Reply:
added explanation in line with dev net update.